aws/role.go

```diff
@@ -130,15 +131,15 @@ func appendBucket(role string, bucket string) string {
 
 	b, err := json.Marshal(rp)
 	if err != nil {
-		fmt.Println(err)
+		log.Error(err.Error())
 	}
 
 	return string(b)
 
 }
 
 func roleError(bucket string, err error) {
-	fmt.Println(err)
+	log.Error(err.Error())
 	fmt.Println(roleMsg)
```

Comment:
@fabioDMFerreira `roleMsg` value is as follows, 
```
We could not create a vmimport Role for you as this account probably does not have the right permissions. Please ask or manually create a vmimport role. For more information see: https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-troubleshooting.html
```

from the value, I think it is mandatory to be displayed because `log.Info` only displayed if user wants verbosity so the one that I log is the error instead.

Reply:
You're right. Let's keep it.